cloudbuild.provision.yaml

```diff
@@ -188,4 +191,4 @@ steps:
       - 'CLOUDSDK_CONTAINER_CLUSTER=$_GKE_CLUSTER_NAME'
 
 options:
-  dynamic_substitutions: true
\ No newline at end of file
+  dynamic_substitutions: true
```

Comment:
I don't know why this appears to be changed - this line is present in the original file and the changed file.